Framework/DataHandling/test/LoadEventNexusTest.h

```diff
@@ -149,8 +149,9 @@ class LoadEventNexusTest : public CxxTest::TestSuite {
 
 public:
   void test_load_event_nexus() {
-    const std::string file = "SANS2D_ESS_example.nxs";
+    const std::string file = "/Users/spu92482/Downloads/V20_example_5.nxs";
```

Comment:
This hard-coded path should not be there.
However, maybe a good idea to add a test where a `V20_example` is being loaded?

Reply:
Thanks Neil. Removing now.